lib/actions/reviewer-survey/index.js

```diff
@@ -0,0 +1,17 @@
+import { laboratoriaAPIAction } from '../helpers';
+
+export const getLastVersion = () => laboratoriaAPIAction({
+  type: 'LAST_VERSION',
```

Comment:
Shouldn't this be `REVIEWER_SURVEY_LATEST_VERSION` instead? same for the method name: `getLatestVersion`.